vmdb/spec/requests/api/tags_spec.rb

```diff
@@ -7,32 +7,34 @@
 
   include Rack::Test::Methods
 
-  before(:each) do
-    init_api_spec_env
+  let(:zone)         { FactoryGirl.create(:zone, :name => "api_zone") }
+  let(:miq_server)   { FactoryGirl.create(:miq_server, :guid => miq_server_guid, :zone => zone) }
+  let(:ems)          { FactoryGirl.create(:ems_vmware, :zone => zone) }
+  let(:host)         { FactoryGirl.create(:host) }
 
-    @zone       = FactoryGirl.create(:zone, :name => "api_zone")
-    @miq_server = FactoryGirl.create(:miq_server, :guid => miq_server_guid, :zone => @zone)
-    @ems        = FactoryGirl.create(:ems_vmware, :zone => @zone)
-    @host       = FactoryGirl.create(:host)
+  let(:tag1)         { {:category => "department", :name => "finance", :path => "/managed/department/finance"} }
```

Comment:
In the future, maybe we could have `let(:classification1) { Classification.classify(vm2, "department", "name") }`

and if we need to reference the path it would be something like `classification_path(classification1)`